derive/src/lib.rs

```diff
@@ -71,6 +66,10 @@ pub fn eth_abi(args: TokenStream, input: TokenStream) -> TokenStream {
 							extern crate parity_hash;
 							extern crate pwasm_ethereum;
 							extern crate pwasm_abi;
+							extern crate pwasm_std;
+							use pwasm_std::hash::{Address, H256};
+							use pwasm_std::Vec;
```

Comment:
would be nice to include all of these under some namespace module in pwasm_abi
and just add `use pwasm_abi::types::*` here

since it requires implementing `pwasm_abi::AbiType` for the said type anyway, this would be senseful